server/plugin/plugin.go

```diff
@@ -250,6 +275,149 @@ func (p *Plugin) OnDeactivate() error {
 	return nil
 }
 
+func (p *Plugin) getPostPropsForReaction(reaction *model.Reaction) (orgRepo []string, id float64, objectType string, ok bool) {
+	post, err := p.client.Post.GetPost(reaction.PostId)
+	if err != nil {
+		p.API.LogError("Error fetching post for reaction", "error", err.Error())
```

Comment:
```suggestion
		p.API.LogError("Error fetching post for reaction", "Error", err.Error())
```
check everywhere. Can skip this if we need to keep it in sync with existing code

Reply:
keep it in sync with existing code

Reply:
```suggestion
		p.API.LogDebug("Error fetching post for reaction", "error", err.Error())
```